pipeline-scripts/mastervertical.groovy

```diff
@@ -21,75 +21,68 @@ stage('mastervertical_scale_test') {
                         sh "cat ${property_file_name}"
 			def mastervertical_properties = readProperties file: property_file_name
-			def jump_host = mastervertical_properties['JUMP_HOST']
-			def user = mastervertical_properties['USER']
-			def tooling_inventory_path = mastervertical_properties['TOOLING_INVENTORY']
-			def clear_results = mastervertical_properties['CLEAR_RESULTS']
-			def move_results = mastervertical_properties['MOVE_RESULTS']
-			def containerized = mastervertical_properties['CONTAINERIZED']
-			def use_proxy = mastervertical_properties['USE_PROXY']
-			def proxy_user = mastervertical_properties['PROXY_USER']
-			def proxy_host = mastervertical_properties['PROXY_HOST']
-			def projects = mastervertical_properties['PROJECTS']
-			def setup_pbench = mastervertical_properties['SETUP_PBENCH']
-			def first_run = mastervertical_properties['FIRST_RUN_PROJECTS']
-			def second_run = mastervertical_properties['SECOND_RUN_PROJECTS']
-			def third_run = mastervertical_properties['THIRD_RUN_PROJECTS']
-			def token = mastervertical_properties['GITHUB_TOKEN']
-			def repo = mastervertical_properties['PERF_REPO']
-			def server = mastervertical_properties['PBENCH_SERVER']	
-			def mode = mastervertical_properties['MODE']
-
-			// debug info
-			println "JUMP_HOST: '${jump_host}'"
-			println "USER: '${user}'"
-			println "TOOLING_INVENTORY_PATH: '${tooling_inventory_path}'"
-			println "CLEAR_RESULTS: '${clear_results}'"
-			println "MOVE_RESULTS: '${move_results}'"
-			println "CONTAINERIZED: '${containerized}'"
-			println "PROXY_USER: '${proxy_user}'"
-			println "PROXY_HOST: '${proxy_host}'"
-			println "PROJECTS: '${projects}'"
-			println "TOKEN: '${token}'"
-			
-			// copy the parameters file to jump host
-			sh "git clone https://${token}@${repo} ${WORKSPACE}/perf-dept && chmod 600 ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf"
-			sh "scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf ${property_file_name} root@${jump_host}:/root/properties"
+			def cluster_user = mastervertical_properties['CLUSTER_USER']
+			def cluster_password = mastervertical_properties['CLUSTER_PASSWORD']
+			def cluster_api_url = mastervertical_properties['CLUSTER_API_URL']
+			def sshkey_token = mastervertical_properties['SSHKEY_TOKEN']
+			def orchestration_host = mastervertical_properties['ORCHESTRATION_HOST']
+			def orchestration_user = mastervertical_properties['ORCHESTRATION_USER']
+			def workload_image = mastervertical_properties['WORKLOAD_IMAGE']
+			def workload_job_node_selector = mastervertical_properties['WORKLOAD_JOB_NODE_SELECTOR']
+			def workload_job_taint = mastervertical_properties['WORKLOAD_JOB_TAINT']
+			def workload_job_privileged = mastervertical_properties['WORKLOAD_JOB_PRIVILEGED']
+			def kubeconfig_file = mastervertical_properties['KUBECONFIG_FILE']
+			def pbench_instrumentation = mastervertical_properties['PBENCH_INSTRUMENTATION']
```

Comment:
This var is not defined in the jjb however will come into play once https://github.com/openshift-scale/workloads/pull/61 is merged.